addons/realisticnames/stringtable.xml

```diff
@@ -3363,12 +3365,14 @@
         </Key>
         <Key ID="STR_ACE_RealisticNames_optic_dms">
             <English>Burris XTR II</English>
+            <German>Burris XTR II </German>
             <Chinese>Burris XTR II</Chinese>
             <Chinesesimp>Burris XTR II</Chinesesimp>
             <Japanese>Burris XTR II</Japanese>
         </Key>
         <Key ID="STR_ACE_RealisticNames_optic_dms_ghex">
             <English>Burris XTR II (Green Hex)</English>
+            <German>Burris XTR II (Grün Hex.) </German>
```

Comment:
Leerzeichen zu viel

Reply:
And the period before the closing parenthesis.

Reply:
`Grünes Hex`